core/state/journal.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2016 The go-ethereum Authors
+// Copyright 2024 the go-ethereum Authors
```

Comment:
```suggestion
// Copyright 2024 The go-ethereum Authors
```
Don't know what to do here, should we keep the original date or the new one. I think the new one is okay, since the content of the file is significantly changes.